sdk/src/beta9/abstractions/output.py

```diff
@@ -43,29 +95,101 @@ def __init__(self, *, path: Union[Path, str]) -> None:
         self.id: Optional[str] = None
         self._stub: Optional[OutputServiceStub] = None
 
+    def __del__(self) -> None:
```

Comment:
Does this get called between task invocations? (like after handler exits and Output obj goes out of scope?)

Reply:
Yes it appears so.

Reply:
Tested with endpoint and task_queue. `__del__` is called after every task invocation.